src/config/default-app-config.ts

```diff
@@ -205,7 +205,11 @@ export class DefaultAppConfig implements AppConfig {
     // Limit for years to display using jumps of five years (current year - fiveYearLimit)
     fiveYearLimit: 30,
     // The absolute lowest year to display in the dropdown (only used when no lowest date can be found for all items)
-    defaultLowerLimit: 1900
+    defaultLowerLimit: 1900,
+    // The number of entries in a paginated browse results list.
+    // Rounded to the nearest size in the list of selectable sizes on the
+    // settings menu.  See pageSizeOptions in 'pagination-component-options.model.ts'.
+    pageSize: 20
```

Comment:
@mwoodiupui : Can we copy this new config over to our `src/config/config.example.yml`?  That way it's shown alongside the other browse options here: https://github.com/DSpace/dspace-angular/blob/main/config/config.example.yml#L170-L176

Beyond this small change, I think this PR looks good!

Reply:
Done.